types/instance.d.ts

```diff
@@ -23,7 +23,7 @@ export interface FastifyInstance<
   prefix: string;
   log: Logger;
 
-  addSchema(schema: FastifySchema): FastifyInstance<RawServer, RawRequest, RawReply>;
+  addSchema(schema: unknown): FastifyInstance<RawServer, RawRequest, RawReply>;
```

Comment:
Is there test for this?

Reply:
No, but I'm not sure how to test this. This method accepts arbitrary ajv schemas and causes no typings side-effects.

Reply:
Is this method exposed? I think TSD test would fail if resulting TS code is uncompilable, so you could just call it with non-FastifySchema object literal in test and call it a day.

Reply:
It's exposed. Good idea, thanks! Amended commit & repushed. I created a new file for instance tests: instance.test-d.ts. There was fastify.test-d.ts, but it's focused on testing fastify() builder method.

Reply:
Removed unused imports & repushed again.